contributing/code_of_conduct/care_team.rst

```diff
@@ -1,30 +1,30 @@
-Enforcement Team
-================
+CARE Team
+=========
 
 Our Pledge
 ----------
 
-In the interest of fostering an open and welcoming environment, the enforcement team
-pledge to ensure that the spirit of the :doc:`Code of Conduct </contributing/code_of_conduct/index>`
+In the interest of fostering an open and welcoming environment, the CoC Active Response Ensurers, or CARE,
+pledged to ensure that the spirit of the :doc:`Code of Conduct </contributing/code_of_conduct/index>`
```

Comment:
imho this should stay as `pledge` as its ongoing .. ie. not just in the past

Reply:
The sentence reads a bit odd (I'm not a native English speaker after all), I was in the assumption it was past tense or:
> the CoC Active Response Ensurers, or CARE, pledge _**is**_ to ensure that the spirit of

Should I add the "is"?

Reply:
with this change it sounds more like this pledge is externally defined .. like the pledge of an organization, rather than the individual members. the intention is that this is the "active" pledge of all members of this team.

but if others also think this reads too bumpy, then I can also accept a change to `pledge is`